src/util/native_fs_utils.js

```diff
@@ -520,7 +520,8 @@ async function get_fs_context(nsfs_account_config, fs_backend) {
         uid: (account_ids_by_dn && account_ids_by_dn.uid) ?? nsfs_account_config.uid,
         gid: (account_ids_by_dn && account_ids_by_dn.gid) ?? nsfs_account_config.gid,
         warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
-        backend: fs_backend
+        backend: fs_backend,
+        ...(nsfs_account_config.supplemental_groups && {supplemental_groups: nsfs_account_config.supplemental_groups})
```

Comment:
because napi doesn't accept undefined value for array. so this value needs to be either a list or doesn't exist at all. napi accepts undefined for int type, if the other values can even be undefined

Reply:
Could you add a comment about this?
Maybe an if can be more readable? (not sure)